packages/authentication/docs.json

```diff
@@ -1,13 +1,17 @@
 {
-    "content": [
-        "index.ts",
-        "src/index.ts",
-        "src/decorator.ts",
-        "src/strategy-adapter.ts"
-    ],
-    "codeSectionDepth": 4,
-    "assets": {
-      "/": "/docs",
-      "/docs": "/docs"
-    }
+  "content": [
+    "index.ts",
+    "src/index.ts",
+    "src/decorators/authenticate.ts",
+    "src/providers/auth-metadata.ts",
+    "src/providers/authenticate.ts",
+    "src/auth-component.ts",
+    "src/keys.ts",
+    "src/strategy-adapter.ts"
+  ],
+  "codeSectionDepth": 4,
+  "assets": {
+    "/": "/docs",
+    "/docs": "/docs"
```

Comment:
While you are editing this part, could you please remove either entire "assets" section, or at least this "/docs" entry, since we are not using any assets and there is no "/docs" directory in this package.

Same comment applies to other `docs.json` files below.

Reply:
Yeah I agree it doesn't need to be there based on my understanding of https://github.com/strongloop/strong-docs#assets.